core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java

```diff
@@ -58,6 +58,7 @@ public MfClientHttpRequestFactoryImpl(final int maxConnTotal, final int maxConnP
 
     private static CloseableHttpClient createHttpClient(final int maxConnTotal, final int maxConnPerRoute) {
         final HttpClientBuilder httpClientBuilder = HttpClients.custom().
+                disableCookieManagement().
```

Comment:
I have a question about that: could this have an impact on proxy usage (i.e. when mapfish-print needs to use a proxy to get data from external servers)?

Reply:
No it shouldn't...